tools/wheels/upload_wheels.sh

```diff
@@ -0,0 +1,62 @@
+# Copied from numpy version
+# https://github.com/numpy/numpy/blob/main/tools/wheels/upload_wheels.sh
+
+set_travis_vars() {
```

Comment:
I lifted this script from the numpy repo. My default process is to not modify things, just in case we need to copy again, and can't remember any modifications done to the script. But it's not problem to delete it.